app/src/main/java/cm/aptoide/pt/app/view/AppViewFragment.java

```diff
@@ -1673,14 +1678,21 @@ private SpannableString formatAppCoinsRewardMessage(String rewardValue) {
     return appBought;
   }
 
-  private void showAppcInfo(boolean hasAdvertising, boolean hasBilling, String appcReward,
-      double fiatReward, String fiatCurrency) {
+  private void showAppcInfo(boolean hasAdvertising, boolean hasBilling, double appcReward,
```

Comment:
Did it actually get approved to be "There are only X rewards available." and not anything else in the FOMO message?

Reply:
Yes and I think this is already in zeplin. Also, if you check dev, you'll see a commit from Carlos changing the text for that string resource too.